Bottom: 78c31b08a549970b362ea03342bed3f8d72bd18b
Top:    aba0b53500ae9d9eebf54fbd4f80592437c13c60
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-02-18 20:37:43 +0000

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 72b6903..74f26d7 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -184,7 +184,7 @@ DrawStrippedText (hidGC gc, ElementType *Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && hid_draw_is_gui (hid_draw)) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& hid_draw_is_gui (hid_draw)*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   ghid_set_lock_effects (Output.fgGC, (AnyObjectType *)element);
